config/install/field.storage.oe_agenda_session.oe_session_details.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - oe_agenda
+id: oe_agenda_session.oe_session_details
+field_name: oe_session_details
+entity_type: oe_agenda_session
+type: string_long
```

Comment:
Following discussions in some recent meetings, I think that having a field that doesn't allow markup will be too restrictive in the future. We should use text formatted long instead, and it will be the project that can configure which markup to show.

Reply:
Done in [06279bc](https://github.com/openeuropa/oe_agenda/pull/4/commits/06279bc13fce322c774e111f5d00a43751ea9d44)